pkg/didcomm/protocol/issuecredential/service.go

```diff
@@ -295,6 +305,12 @@ func (s *Service) handle(md *metaData) error {
 		return fmt.Errorf("failed to persist state %s: %w", stateName, err)
 	}
 
+	for _, credential := range md.credentials {
+		if err := s.verifiable.SaveCredential(credential.ID, credential); err != nil {
```

Comment:
Why does service need to store the credential directly? Shouldn't it be up-to the consumer to decide ?

Reply:
The consumer can decide whether to accept issued credentials or not. If he accepts it we simply store it into the verifiable store. 

Why does service need to store the credential directly? - it is according to the [diagram](https://github.com/hyperledger/aries-rfcs/blob/master/features/0036-issue-credential/credential-issuance.png).
NOTE: See when the Holder receives an issue-credential message.

Reply:
Ideally we need to store it with a friendly name, so that the user can search easily. May be we need to take that input from the user while accepting.

Reply:
I skipped this part with a friendly name. I did not know what to put there so, I decided to use ID. Get a friendly name from the user sounds good. But as I understand we can receive several credentials.  Not sure whether it is ok to provide an option for that e.g `(credentialNames ...string)`. Looks like the user should count credentials a provide the same number of names. What if names did not match the count of credentials? Should it be an error or we can use ID? 

What do you think about the name? Is it ok to generate a name e.g based on the ID or any other field from the credential payload?
Or provide a function to e.g. `WithCustomName(credentialNames ...string) Opt`

Reply:
@soluchok fow now i would provide `WithFriendlyNames(names ...string)`

Reply:
Ok

Reply:
Done!